Use the Python environement from EB in the Guroby Python shell
author: Alex Domingo (Vrije Universiteit Brussel), ported to Gurobi 11.0.0 by Kenneth Hoste (HPC-UGent)
--- gurobi1100/linux64/bin/gurobi.sh.orig	2024-02-12 19:31:52.884375084 +0100
+++ gurobi1100/linux64/bin/gurobi.sh	2024-02-12 19:37:00.624272524 +0100
@@ -7,10 +7,6 @@
   echo
 fi
 
-PATH=$GUROBI_HOME/bin:$PATH;export PATH
-LD_LIBRARY_PATH=$GUROBI_HOME/lib:$LD_LIBRARY_PATH;export LD_LIBRARY_PATH
-PYTHONHOME=$GUROBI_HOME;export PYTHONHOME
+PYTHONSTARTUP=$EBROOTGUROBI/lib/gurobi.py;export PYTHONSTARTUP
 
-PYTHONSTARTUP=$PYTHONHOME/lib/gurobi.py;export PYTHONSTARTUP
-
-$PYTHONHOME/bin/python3.11 "$@"
+$EBROOTPYTHON/bin/python "$@"
